defaults/main.yml

```diff
@@ -51,7 +51,7 @@ vault_ui: "{{ lookup('env', 'VAULT_UI') | default(false, true) }}"
 vault_port: 8200
 vault_node_name: "{{ inventory_hostname_short }}"
 vault_main_config: "{{ vault_config_path }}/vault_main.hcl"
-vault_backend: vault_backend_consul.j2
+vault_backend_consul: vault_backend_consul.j2
```

Comment:
This role currently supports both the Consul and File storage backends (and potentially others in the future); not sure this change makes sense.